File: applied/5_top_of_the_world/answer/1.java
// 1つの数字を入力し、その数字が0以下になるまで5を引いていき、最後に5を引いた回数を出力するプログラムを作成しよう

import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        int count = 0;
        int input;
        
        for (input = scan.nextInt(); input > 0; input -= 5, count++) ;
        
        System.out.printf("%d\n", count);
    }
}

File: applied/05_top_of_the_world/answer/1.c
// 1つの数字を入力し、その数字が0以下になるまで5を引いていき、最後に5を引いた回数を出力するプログラムを作成しよう

#include <stdio.h>
int main(void){
    int input, count;
    scanf("%d", &input);
    
    for (count = 0; input > 0; count++) {
        input -= 5;
    }
    
    printf("%d\n", count);
}
